lib/drivers/docker/index.js

```diff
@@ -0,0 +1,221 @@
+var sCtl = require('strong-control-channel/client');
+var Deployment = require('./deployment');
+var EventEmitter = require('events').EventEmitter;
+var fmt = require('util').format;
+var inherits = require('util').inherits;
+var sdr = require('strong-docker-run');
+var _ = require('lodash');
+
+module.exports = exports = DockerDriver;
+
+function DockerDriver(opts) {
+  if (!opts.baseDir || !opts.console || !opts.server) {
+    throw Error('Requires baseDir, console, and server');
+  }
+  if (!(this instanceof DockerDriver)) {
+    return new DockerDriver(opts);
+  }
+  EventEmitter.call(this);
+
+  // XXX(rmg) can I put temp files here?
```

Comment:
I guess so, but who will clean them up? The nice thing about /tmp is it gets cleaned for us. But as long as you use a tmp prefix, you can write what you want in base.